.github/workflows/ci.yaml

```diff
@@ -0,0 +1,58 @@
+name: FINOS Legend Bundle Test Jobs
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+  schedule:
+    # Runs everyday at 00:00. (see https://crontab.guru)
+    - cron: "0 0 * * *"
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: python3 -m pip install tox
+
+      - name: Run linters
+        run: tox -vve lint
+
+  integration-test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: python -m pip install tox
+
+      - name: Setup operator environment
+        # TODO: change this to charmed-kubernetes/actions-operator@main once
+        # the following issue is addressed:
+        # https://github.com/charmed-kubernetes/actions-operator/issues/32
```

Comment:
We should use `charmed-kubernetes/actions-operator@main`, since the issue mentioned there was solved in [here](https://github.com/charmed-kubernetes/actions-operator/pull/46). This will fix the problem with the `setup operator environment`

Reply:
Thanks. This was solved on the ``edge`` branch before [1], so it's worth cherry-picking here as well.

[1] https://github.com/finos/legend-juju-bundle/pull/33